Test/TicTacToe/TicTacToe/TicTacToeGame/Game.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace TicTacToeGame;
+
+/// <summary>
+/// Class representing TicTacToe (3 x 3) game.
+/// </summary>
+public class Game
+{
+    // Found three same value in the row
+    public bool IsGameOver { get; private set; } = false;
+
+    // No place to move
+    public bool Draw { get; private set; } = false;
+
+    // Who made previous move
+    public Player movedPlayer = Player.Second;
```

Comment:
```suggestion
    public Player MovedPlayer { get; set; } = Player.Second;
```
Принято, чтобы поля были приватными, а свойства публичными.